oci/layout/oci_dest.go

```diff
@@ -53,6 +53,12 @@ func (d *ociImageDestination) ShouldCompressLayers() bool {
 	return false
 }
 
+// CopyForeignLayers returns true iff foreign layers in manifest should be actually
+// uploaded to the image destination, false otherwise.
+func (d *ociImageDestination) CopyForeignLayers() bool {
+	return false
+}
```

Comment:
Why is this `true` for `dir:` but `false` for `oci:`? At a first glance it seems that the two should behave similarly.